### PR TITLE
Fixed unclear instruction

### PR DESCRIPTION
**Pull request by isaac-s**
Literals should be reversed...

### CODE REVIEW

**Comment by nir0s** [comment]
Please fix the test also.
https://github.com/cloudify-cosmo/cloudify-plugin-template/blob/master/plugin/tests/test_plugin.py#L49

**Comment by isaac-s** [comment]
@nir0s done.

**Comment by barakm** [comment]
@nir0s can we merge this?

File: plugin/tests/blueprint/blueprint.yaml [hunk 1]
```diff
@@ -39,4 +39,5 @@ outputs:
     # example output the could be used to simplify assertions by test
     test_output:
         description: an output for the test
-        value: { get_attribute: [test_node_template, value_of_some_property]}
+        value: { get_attribute: [test_node_template, some_property
+        ]}
```
Comment (nir0s) [inline plugin/tests/blueprint/blueprint.yaml]: ehm.. just wondering.. why the newline?

Comment (isaac-s) [inline plugin/tests/blueprint/blueprint.yaml]: Beats the heck out of me. Did I do this?!

I'll fix.

**Pull request merged**